src/test/java/org/kie/processmigration/integration/HealthStatusIT.java

```diff
@@ -16,42 +16,26 @@
 
 package org.kie.processmigration.integration;
 
-import io.quarkus.test.junit.QuarkusTest;
-import io.quarkus.test.junit.TestProfile;
 import io.restassured.response.ResponseBodyExtractionOptions;
 import org.apache.http.HttpStatus;
 import org.junit.jupiter.api.Test;
-import org.kie.processmigration.model.KieServerConfig;
-import org.kie.processmigration.service.KieService;
-import org.kie.processmigration.test.Profiles;
-
-import javax.inject.Inject;
 
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 
-@QuarkusTest
-@TestProfile(Profiles.KieServerIntegrationProfile.class)
-class HealthStatusIT {
-
-    @Inject
-    KieService service;
+class HealthStatusIT extends AbstractBaseIT {
 
     @Test
-    void testLivenessHealthChecks() {
+    void testLivelinessHealthChecks() {
```

Comment:
```suggestion
    void testLivenessHealthChecks() {
```

Reply:
Since this is a cherry-pick from another [commit](https://github.com/kiegroup/process-migration-service/pull/19/commits/35c6cbd4d19022b9aef792369fa0359ff759b727) in main branch , I'd prefer to leave it as is